terraformV2/gke/db.tf

```diff
@@ -0,0 +1,51 @@
+provider "random" {
+  version     = "~> 2.1"
+}
+
+resource "random_id" "travela-db-name" {
+  byte_length = 8
+}
+
+resource "random_id" "travela-db-user" {
+  byte_length = 8
+}
+
+resource "random_id" "travela-db-user-password" {
+  byte_length = 16
+}
+
+resource "google_sql_database_instance" "travela-db-instance" {
+  region           = "${var.region}"
+  database_version = "POSTGRES_9_6"
+  name             = "${var.environment}-travela-db-instance-${replace(lower(random_id.travela-db-name.b64), "_", "-")}"
+  project          = "${var.project}"
```

Comment:
Please remove `project` key from all the resources.

Reply:
Done